File: 140407/forTest0/test0.php
<?php
// test0.php //
function test($i, $j)
{
	return ($i * $j);
}

function main()
{
	$i = 2;
	$j = 3;
	$k = test($i, $j);
	if (k > 5) {
		echo ">5\n";
	} else {
		echo "<=5\n";
	}
}

main();
?>

File: 140407/forTest0/test0.rb
def test(i, j)
	return (i * j)
end

def main()
	i = 2
	j = 3
	k = test(i, j)
	if k > 5
		print ">5\n"
	else
		print "<=5\n"
	end
end

main()